packages/clients/textLocator/package.json

```diff
@@ -10,13 +10,15 @@
     "url": "https://github.com/Dataport/polar.git",
     "directory": "packages/clients/textLocator"
   },
+  "main": "dist/client-text-locator.js",
   "files": [
     "dist/**/**.*",
     "CHANGELOG.md"
   ],
   "scripts": {
     "postversion": "npm run build",
-    "build": "rimraf dist && vite build && bash ./scripts/injectFlag.sh"
+    "build": "rimraf dist && vite build && copyfiles -f src/html/**/* dist",
+    "build:test": "rimraf dist && vite build && bash ./scripts/injectFlag.sh"
```

Comment:
This seems to be the only client that has a `build:test` script.

Is the `injectFlag.sh` actually still needed? Refresh my memory please :D

Reply:
Forgot to clean up. https://github.com/Dataport/polar/pull/168/commits/dd6ae5368f95b2eac4852e2d4951f1fd53d79b57 Apparently `injectFlag.sh` is not needed as `build:test` was called nowhere.